eng/CodeCoverage.config

```diff
@@ -5,8 +5,10 @@
 
     <ModulePaths>
       <Include>
-        <ModulePath>.*Microsoft.*\.dll$</ModulePath>
-        <ModulePath>.*System.*\.dll$</ModulePath>
+        <!-- <ModulePath>.*Microsoft.AspNetCore\.[^\\]*\.dll$</ModulePath>
+        <ModulePath>.*Microsoft.Extensions\.[^\\]*\.dll$</ModulePath>
+        <ModulePath>.*Microsoft.Gen\.[^\\]*\.dll$</ModulePath> -->
+        <ModulePath>.*System\.Cloud\.([^\\]*)\.dll$</ModulePath>
```

Comment:
Super small NIT, but using `+` quantifier instead of `*` will be slightly more performant. Also, no need to add the parenthesis as you don't need that capture group.